pyproject.toml

```diff
@@ -47,6 +47,7 @@ coiled = ">=1.27.2"
 dask = ">=2024.6.0"
 colorama = ">=0.4.6"
 s3path = ">=0.5.7"
+matplotlib = "^3.9.1"
```

Comment:
hum, i guess this should be a dev dependencies, and then install it with ```poetry install ---with dev```. will chat about it

Reply:
that is ok, happy to move it to dev